### PR TITLE
Add --sort for sorting dump/latex output

### PR DESCRIPTION
**Pull request by zaucker**
I wonder if it should be `@order := ...`, but then the doing the `.reverse` inside the ternary wouldn't work I suppose.

### CODE REVIEW

File: lib/Agrammon/UI/CommandLine.pm6 [hunk 1]
```diff
@@ -57,16 +57,16 @@ multi sub MAIN('run', ExistingFile $filename, ExistingFile $input, Str $tech-fil
 }
 
 #| Dump model
-multi sub MAIN('dump', ExistingFile $filename, Str :$variants = 'SHL') is export {
-    say chomp dump-model $filename.IO, $variants;
+multi sub MAIN('dump', ExistingFile $filename, Str :$variants = 'SHL', Str :$sort = 'model') is export {
```
Comment (jnthn) [inline lib/Agrammon/UI/CommandLine.pm6:60]: Maybe a `subset` type (or just anonymous `where`) to catch any typos?

File: lib/Agrammon/UI/CommandLine.pm6 [hunk 2]
```diff
@@ -57,16 +57,16 @@ multi sub MAIN('run', ExistingFile $filename, ExistingFile $input, Str $tech-fil
...
+    say chomp dump-model $filename.IO, $variants, $sort;
 }
 
-multi sub MAIN('latex', ExistingFile $filename, Str $tech-file?, Str :$variants = 'SHL') is export {
-    latex $filename.IO, $tech-file, $variants;
+multi sub MAIN('latex', ExistingFile $filename, Str $tech-file?, Str :$variants = 'SHL', Str :$sort = 'model') is export {
```
Comment (jnthn) [inline lib/Agrammon/UI/CommandLine.pm6:64]: Given it's repeated in a number of places, probably the `subset` type is better.

**Review by jnthn — COMMENTED** [review]
> I wonder if it should be @order := ..., but then the doing the .reverse inside the ternary wouldn't work I suppose.

Correct, you can't bind a `Seq` to an `@` sigil; you could use an sigilless symbol though (`my \order = ...`), which will retain the laziness of the `reverse` operation.

File: lib/Agrammon/UI/CommandLine.pm6 (hunk 2 above)
Comment (zaucker) [reply lib/Agrammon/UI/CommandLine.pm6:64]: Done.

**Comment by zaucker** [comment]
Also changed to `\order`, forgot about those.

**Review by jnthn — APPROVED** [review]

**Pull request merged**